src/auto-evo/simulation/SimulationCache.cs

```diff
@@ -410,12 +412,18 @@ public void Clear()
 
             if (organelle.Definition.HasSlimeJetComponent)
             {
-                slimeJetScore += Constants.AUTO_EVO_MUCILAGE_PREDATION_SCORE;
+                if (organelle.Upgrades?.UnlockedFeatures.Contains(SlimeJetComponent.MUCOCYST) ?? false)
```

Comment:
```suggestion
                if (organelle.Upgrades?.UnlockedFeatures.Contains(SlimeJetComponent.MUCOCYST) == true)
```

I prefer this way of writing the nullable bool check.